room_assistance/indico_room_assistance/client/style.scss

```diff
@@ -0,0 +1,32 @@
+// This file is part of the CERN Indico plugins.
+// Copyright (C) 2014 - 2019 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+
+.assistance-occurrences-widget {
+    display: flex;
+    flex-direction: column;
+
+    .occurrence + .occurrence {
+        margin-top: 5px;
+    }
+
+    input ~ input {
+        margin-right: 1em;
+    }
+
+    .occurrence {
+        align-items: center;
+        display: inline-flex;
+
+        .datepicker {
+            width: 40% !important;
+        }
+
+        .timepicker {
+            width: 20% !important;
```

Comment:
I think you're trying to hide the `PM/AM` in the time picker? If that's the case what's the reason behind? this is causing problems in Chrome.
![image](https://user-images.githubusercontent.com/8863238/60523606-4a576580-9ceb-11e9-8ebe-75a5f92565f5.png)

Reply:
Nope I am not trying to hide it. That is a good catch I didn't notice that since I couldn't even see the P like in your case.

Reply:
Hm, IT is a bit strange since I thought I was using 24h clock format there. Are you using different locale or sth?

Reply:
`en-US` AFAIK, `es-ES` as second language. In both Chrome and Firefox I get 12h clock.

Reply:
Increased the width of the time field.

Reply:
Looks good now! :+1: